SPEC.md

```diff
@@ -633,8 +654,8 @@ JSON Schema for the Container Runtime Manifest (container manifest), conforming
     * **annotations** (list of objects, optional) arbitrary metadata appended to the app. The annotation objects must have a *name* key that has a value that is restricted to the [AC Name](#ac-name-type) formatting and *value* key that is an arbitrary string). Annotation names must be unique within the list. These will be merged with annotations provided by the image manifest when queried via the metadata service; values in this list take precedence over those in the image manifest.
 * **volumes** (list of objects, optional) list of volumes which should be mounted into each application's filesystem
     * **name** (string, required) used to map the volume to an app's mountPoint at runtime. (restricted to the AC Name formatting)
-    * **kind** (string, required) either "empty" or "host". "empty" fulfills a mount point by ensuring the path exists (writes go to container). "host" fulfills a mount point with a bind mount from a **source**.
-    * **source** (string, required if **kind** is "host") absolute path on host to be bind mounted into the container under a mount point.
+    * **kind** (string, required) either "empty" or "host". "empty" fulfills a mount point by ensuring the path exists (writes go to pod). "host" fulfills a mount point with a bind mount from a **source**.
```

Comment:
Both "writes go to container" and "writes go to pod" feel wrong here now. Each app in a pod has its own chroot. "Host" mounts would be shared between apps via the bind mount in each chroot, but "empty" now seems to imply writes go to each app's respective chroot, not "to pod". With the introduction of pods, would changing "empty" to "pod" make it more clear that this is a shared volume for apps in a pod? Would probably make sense to reword the description too ("path exists" isn't sufficient to imply a shared volume). Leave "empty" as dummy volume per chroot, add "pod" as shared volume?